File: mario-less/mario.c
#include <stdio.h>

int main()
{
    int num;

    // Prompt the user for the pyramid's height

    do
    {
        printf("Height: ");
        scanf("%d", &num);
    }
    while (num <= 0 || num > 8);

    // Print a pyramid of that height

    for (int i = 1; i <= num; i++)
    {

        // Print the rows

        for (int j = num - i, k = 0; k < num; k++)
        {
            if (k >= j)
                printf("#");
            else
                printf(" ");
        }
        printf("\n");
    }
    return 0;
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    num = get_int("Height: ")
    if num > 0 and num < 9:
        break

for i in range(1, num + 1):
    j = num - i
    for k in range(0, num):
        if k >= j:
            print("#", end="")
            if k == num - 1:
                print("  ", end="")
                print("#" * i)
        else:
            print(" ", end="")
